src/core/model_config.proto

```diff
@@ -517,6 +517,62 @@ message ModelSequenceBatching
   repeated ControlInput control_input = 2;
 }
 
+//@@
+//@@.. cpp:var:: message ModelEnsembling
+//@@
+//@@   Model ensembling configuration. These settings control how the inputs
+//@@   and outputs of the models are mapped together.
+//@@
+message ModelEnsembling
+{
+  //@@  .. cpp:var:: message Step
+  //@@
+  //@@     The step specifies a model included in the ensemble and how its
+  //@@     inputs and outputs are mapped to names that is recognizable within
+  //@@     the ensemble.
+  //@@
+  message Step
+  {
+    //@@    .. cpp:var:: string model_name
+    //@@
+    //@@       The name of the model.
+    //@@
+    string model_name = 1;
+
+    //@@  .. cpp:var:: map<string,string> input_alias_map
+    //@@
+    //@@     Map from the inputs specified in the model cofig to aliases
```

Comment:
Map from ensemble tensor name to name of an input tensor on this step's model. The ensemble tensor must have the same datatype and shape as the model input. Each model input must be assigned to one ensemble tensor, but the same ensemble tensor can be assigned to multiple model inputs.